The_Lost_Star/events/activities/expedition_activity/expedition_events.txt

```diff
@@ -0,0 +1,439 @@
+	option = { #Do nothing
+		name = expedition.4002.a
+		scope:activity = {
+			add_activity_log_entry = {
+				key = expedition_warehouse_do_nothing
+				score = 25
+				tags = { random }
+				character = root
+
+				#Effect
+			}
+		}
+		ai_chance = {
+			base = 30
+			ai_value_modifier = {
+			}
+		}
+	}
+
+	option = { #Search for valuables
+		name = expedition.4002.b
+		scope:activity = {
+			add_activity_log_entry = {
+				key = expedition_warehouse_search_valuables
+				score = 25
+				tags = { random good }
+				character = root
+
+				#Effect
+				root = {
+					duel = {
+						skill = stewardship
+						value = 16
+						50 = {
+							desc = expedition.4002.b.success
+							compare_modifier = {
+								value = scope:duel_value
+								multiplier = 3.5
+								min = -25
+							}
+							send_interface_toast = {
+								title = expedition.4002.b.success
+								add_gold = medium_gold_value
+							}
+						}
+						40 = {
+							desc = expedition.4002.b.failure
+							compare_modifier = {
+								value = scope:duel_value
+								multiplier = -3.5
+								min = -25
+							}
+							send_interface_toast = {
+								title = expedition.4002.b.failure
+							}
+						}
+					}
+				}
+			}
+		}
+		ai_chance = {
+			base = 30
+			ai_value_modifier = {
+				ai_greed = 1
+			}
+		}
+	}
+	
+	option = { #Search for information
+		name = expedition.4002.c
+		scope:activity = {
+			add_activity_log_entry = {
+				key = expedition_warehouse_search_information
+				score = 25
+				tags = { random good }
+				character = root
+
+				#Effect
+				root = {
+					duel = {
+						skill = stewardship
+						value = 16
+						50 = {
+							desc = expedition.4002.c.success
+							compare_modifier = {
+								value = scope:duel_value
+								multiplier = 3.5
+								min = -25
+							}
+							send_interface_toast = {
+								title = expedition.4002.c.success
+								expedition_activity_success_change_effect = { CHANGE = increase_medium }
+							}
+						}
+						40 = {
+							desc = expedition.4002.c.failure
+							compare_modifier = {
+								value = scope:duel_value
+								multiplier = -3.5
+								min = -25
+							}
+							send_interface_toast = {
+								title = expedition.4002.c.failure
+							}
+						}
+					}
+				}
+			}
+		}
+		ai_chance = {
+			base = 70
+			ai_value_modifier = {
+				ai_energy = 1
+			}
+		}
+	}
+	
+}
+
+expedition.5001 = {
+	type = activity_event
+	title = expedition.5001.t
+	desc = {
+		desc = expedition.5001.opening
+		desc = expedition.5001.opening.2
+		desc = expedition.5001.opening.3
+	}
+
+	theme = expedition_activity_fuse
+	left_portrait = {
+		character = root
+		animation = thinking
+	}
+	right_portrait = {
+		character = scope:expedition_participant
+		animation = idle
+	}
+	cooldown = { years = 1 }
+	
+
+	immediate = {
+		scope:activity = {
+			random_attending_character = {
+				limit = {
+					NOT = { this = scope:host }
+				}
+				save_scope_as = expedition_participant
+			}
+		}
+	}
```

Comment:
As a note, this will fail if the host is the only person in the expedition
Which will lead to a weird looking event

Not important for this PR, but just a note, you should have a trigger before this to check if any_attending_character exists that is not the scope:host